test/bigtest/interactors/users/header-dropdown-menu.js

```diff
@@ -0,0 +1,12 @@
+import {
+  clickable,
+  interactor,
+  isVisible,
+} from '@bigtest/interactor';
+
+@interactor class HeaderDropdownMenu {
+  clickExportToCSV = clickable('#export-overdue-loan-report');
+  exportBtnIsVisible = isVisible('#export-overdue-loan-report');
+}
+
```

Comment:
@SergiyVSergiyenko, plenty of changes in current module were done! And in tests occurs very simular interactors with same names (like HeaderDropdown, CellInteractor, RowInteractor), I guess that will confuse other developers, when they try to search if current interactor already exists and will find few identical interactors. I think you could rename interactors with more informative way (with clarifying to which page/view current interactor relates), or if that is possible in some cases - to make one reusable interactor without duplications. What do you think?